infra/Pulumi.prod.yaml

```
@@ -4,6 +4,8 @@ config:
   gateway:debeziumDbUser:
     secure: AAABAD3UsX0N4DL09nVRVJZ2uM/6FaENmEOzbwj8XFNfLZxVb9KGuQ==
   gateway:env:
+    SubmitArticleTreshold:
```

This should be camelCase. It is safe to apply this change

```suggestion
    submitArticleTreshold:
```